subsys/bluetooth/controller/Kconfig

```diff
@@ -897,17 +909,6 @@ config BT_CTLR_PA_LNA_GPIOTE_CHAN
 	help
 	  Select the nRF5 GPIOTE channel to use for PA/LNA GPIO feature.
 
-comment "BLE Controller debug configuration"
```

Comment:
Add this "comment" (that acts as separators) back

Reply:
Done